pheweb/serve/data_access/db.py

```diff
@@ -105,6 +115,34 @@ def get_gene_info(self, symbol):
         loc = list(filter( lambda x: x["annotationrelease"]=="109", data["locationhist"]))[0]
         return { "description":data["description"], "summary":data["summary"], "start":data["chrstart"], "stop":loc["chrstop"], "maploc":data["maplocation"]   }
 
+class DrugDao(DrugDB):
+
+    def __init__(self):
+        self.mg = mygene.MyGeneInfo()
```

Comment:
we are already using ensembl for getting gene symbol -> ensembl id . Why introduce additional dependency?